### DISCUSSION TITLE
Bulk payment when clearing unpaid orders- Suggestion

### DISCUSSION THREAD

**Discussion by mbugikp** [Blair2004/NexoPOS#640] [Ideas]
**Describe the bug**
I have a unique but valid suggestion, currently you cannot clear /accept payment for more than one order at the same time, in our case the cashier is not able to clear all the orders within the same day leading to a pile of unpaid orders of many days, there is always a back log of several days, to avoid this backlog we suggest a feature where the cashier can select several unpaid orders belonging to a particular user/waiter then clear/accept payment them at once.

**Comment by Blair2004** [comment]
Hi, 
Thank you. We might consider this on upcoming NexoPOS 4.7.x release.
Moved to suggestions.

**Reply by mbugikp** [reply]
Thank you, much appreciated.

**Comment by mbugikp** [comment]
Hi Blair,
How is this going? I am hoping we can merge now that 4.7.x is out

**Reply by mbugikp** [reply]
@Blair2004 we should revisit this issue, hope we can now merge with 4.7.x
Regards
